### DISCUSSION TITLE
Select fields suggestions or Tags mode: Select to save as string

### DISCUSSION THREAD

**Discussion by Sandilly** [avo-hq/avo#3549] [Feature feedback]
Hello,

My schema created for a form field is a string but for my admin form, I would like the functionality of the tags suggestions where the user can only select one. I see that the tags field has a mode: select, but on save. It saves as an array. So I would have to massage it to turn it into a string on save and on show or have to change the schema to be of type array, which I am not sure if it makes much sense.

Is there such functionality that will address this out the box, if not what is the best approach?

**Comment by Paul-Bob** [comment]
Closing as duplicated of https://github.com/avo-hq/avo/discussions/3548
